.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Deploy
+
+on:
+  push:
+    branches: ["dev", "feature/depoy"]
+    paths: 
+      - ".github/workflows/release.yml"
+  pull_request:
+    branches: ["dev", "feature/depoy"]
+    paths: 
+      - ".github/workflows/release.yml"
+
+jobs:
+  distribute:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: "17.0"
+
+      - name: Setup Flutter
+        uses: subosito/flutter-action@v2
+        with:
+          channel: 'stable'
+      - name: Install Fastlane
+        run: gem install fastlane
+      - name: Configure Keystore
+        run: |
+          echo "$ANDROID_KEYSTORE_FILE" > keystore.jks.b64
+          base64 -d -i keystore.jks.b64 > app/keystore.jks
+          echo "storeFile=keystore.jks" >> keystore.properties
+          echo "keyAlias=$KEYSTORE_KEY_ALIAS" >> keystore.properties
+          echo "storePassword=$KEYSTORE_STORE_PASSWORD" >> keystore.properties
+          echo "keyPassword=$KEYSTORE_KEY_PASSWORD" >> keystore.properties
+        env:
+          ANDROID_KEYSTORE_FILE: ${{ secrets.ANDROID_KEYSTORE_FILE }}
+          KEYSTORE_KEY_ALIAS: ${{ secrets.KEYSTORE_KEY_ALIAS }}
+          KEYSTORE_KEY_PASSWORD: ${{ secrets.KEYSTORE_KEY_PASSWORD }}
+          KEYSTORE_STORE_PASSWORD: ${{ secrets.KEYSTORE_STORE_PASSWORD }}
+
+      - name: Create Google Play Config file
+        run : |
+          echo "$PLAY_CONFIG_JSON" > fastlane_secret_key.json.json.b64
+          base64 -d -i fastlane_secret_key.json.json.b64 > fastlane_secret_key.json.json
+        env:
+          PLAY_CONFIG_JSON: ${{ secrets.PLAY_CONFIG_JSON }}
+      - name: Distribute app to Beta track 🚀
+        run: bundle exec fastlane beta
```

Comment:
Change to
```
fastlane beta
```
Rememeber set `working-directory`

Reply:
Please pay attention to the line 'Remember'

Reply:
Oh I'm sorry. I'll set working-directory